launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml

```diff
@@ -11,13 +11,12 @@
   <arg name="lidar_detection_model" default="centerpoint" description="options: `centerpoint`, `apollo`, `pointpainting`, `clustering`"/>
   <arg name="score_threshold" default="0.35"/>
 
-  <arg name="use_pointcloud_map" default="true" description="use pointcloud map in detection"/>
```

Comment:
@miursh I just remove default value of `use_pointcloud_map` to reduce parameter value confusion. `use_pointcloud_map` value will be defined in https://github.com/autowarefoundation/autoware.universe/blob/9639f8a10b30b96b30b612dadf4882dcf604bd5b/launch/tier4_perception_launch/launch/perception.launch.xml#L59
and it is parsed to pointcloud_map_filter.launch. py at https://github.com/autowarefoundation/autoware.universe/blob/9639f8a10b30b96b30b612dadf4882dcf604bd5b/launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml#L70

Reply:
@miursh I also fixed a bug of "use_pointcloud_map" passing here https://github.com/autowarefoundation/autoware.universe/pull/4999/commits/a0b6c7b595577eaf3f7b9f7725b6e99ea909a60f

Reply:
@badai-nguyen 
I believe we should keep this argument in each launcher file because
1. Is makes hard to understand about the arguments used in a launcher
2. It is difficult to use this by itself because the argument is used within this launcher but is not listed at the beginning of the launcher
However, we should reorganize launchers in another PR.

Reply:
@miursh I understood, I will readd the argument definition at each launch file.